Formula/p/python-rpds-py.rb

```diff
@@ -0,0 +1,34 @@
+class PythonRpdsPy < Formula
+  desc "Python bindings to Rust's persistent data structures"
+  homepage "https://github.com/crate-py/rpds"
+  url "https://files.pythonhosted.org/packages/77/5a/0c82d0ef1322227e8e997dbbd3d4e235383d51c299dbdfd2fed2625971b0/rpds_py-0.10.0.tar.gz"
+  sha256 "e36d7369363d2707d5f68950a64c4e025991eb0177db01ccb6aa6facae48b69f"
+  license "MIT"
+
+  depends_on "maturin" => :build
+  depends_on "rust" => :build
+  depends_on "python@3.11"
+
+  def python
+    "python3.11"
+  end
+
+  def install
+    system python, "-m", "pip", "install", "--prefix=#{prefix}", "--no-deps", "."
```

Comment:
this is still pulling binary build requirements, did you want to update `maturin` to include the python lib? then this can just use `std_pip_args`

Reply:
Just needs a rebase after #141038